### PR TITLE
Feature/hub 442 add keyboard support to accordion elements

### PR DESCRIPTION
**Pull request by janipalsamaki**

### CODE REVIEW

File: themes/uhsg_theme/js/accordion.js [hunk 1]
```diff
@@ -5,6 +5,12 @@
       $('.accordion__title', context).once().on('click', function () {
         $(this).toggleClass('is-active').next('.accordion__content').toggleClass('visually-hidden');
       });
+      $('.accordion__item', context).once().on('keypress', function (event) {
+        if (event.keyCode != 9) {
```
Comment (mikaelkundert) [inline themes/uhsg_theme/js/accordion.js]: Can we comment what `event.keyCode` `9` means? That's `Tab` I guess?

File: themes/uhsg_theme/css/style.css [hunk 2]
```diff
@@ -8795,6 +8795,11 @@ input[type="text"].search-form-large__input {
   margin-bottom: 0;
 }
 
+.accordion__item:focus:focus {
+  outline: currentColor thin dashed;
+  outline-offset: 3px;
+}
```
Comment (mikaelkundert) [inline themes/uhsg_theme/css/style.css]: I think we should make the focus styling bit more visible. I rarely see a difference.

Screenshot of comparison between unfocused and focused accordion styling

Comment (mikaelkundert) [reply themes/uhsg_theme/css/style.css]: Disregard my comment. I did not restart caches properly after switching the branch.

File: themes/uhsg_theme/js/accordion.js (hunk 1 above)
Comment (janipalsamaki) [reply themes/uhsg_theme/js/accordion.js]: Added comment.

**Pull request merged**